main/resource-access.bs

```diff
@@ -95,6 +95,44 @@ Behaviour pertaining to authorization MUST proceed this optional redirect
 
 ## Linked Data ## {#linked-data}
 
+### Storage ### {#storage}
+
+When a servers supports a data pod, it MUST provide one or more storages
+(`pim:Storage`). A storage is the root container for all of its contained
+resources (see [[#resource-containment]]).
+
+When a server supports multiple storages, the URIs MUST be allocated to
+non-overlapping space.
+
+Servers exposing the storage resource MUST advertise by including the HTTP
+`Link` header with `rel="type"` targeting
+`http://www.w3.org/ns/pim/space#Storage` when responding to request URI.
+
+Clients can determine a resource is of type storage by making an HTTP `HEAD`
+or `GET` request on the target URL, and check for the `Link` header with
+`rel="type"` targeting `http://www.w3.org/ns/pim/space#Storage`.
+
+Clients can determine the storage of a resource by moving up the URI path
+hierarchy until the response includes a `Link` header with `rel="type"`
+targeting `http://www.w3.org/ns/pim/space#Storage`. Clients may check the root
+path of a URI for the storage claim at any time.
+
+Clients can discover a storage by making an HTTP `GET` request on the target
+URL to retrieve an RDF representation [[!RDF11-CONCEPTS]], whose encoded RDF
+graph contains a relation of type `http://www.w3.org/ns/pim/space#storage`.
+The object of the relation is the storage (`pim:Storage`).
```

Comment:
What does the subject mean?

Reply:
Leaving it alone for now.. pim-space doesn't define a domain. Current practice is a profile's storage but we did use it for workspaces as well.